ReText/__init__.py

```diff
@@ -32,6 +32,13 @@
 	settings = QSettings(QSettings.Format.IniFormat, QSettings.Scope.UserScope,
 		'ReText project', 'ReText')
 
+cache = QSettings('ReText project', 'ReText cache')
```

Comment:
Let's name it just `cache.conf`, to make it more friendly to people who open files from terminal.

```suggestion
cache = QSettings('ReText project', 'cache')
```